### DISCUSSION TITLE
Is it not possible to forward the message again to the same chat using the id of the message?

### DISCUSSION THREAD

**Discussion by prabathLK** [WhiskeySockets/Baileys#381] [General Questions] [answered]
Is it not possible to forward the message again to the same chat using the id of the message?

**Comment by Riders004** [comment, marked as answer]
> Is it not possible to forward the message again to the same chat using the id of the message?

Maybe yes

**Comment by shizothetechie** [comment]
> Is it not possible to forward the message again to the same chat using the id of the message?

Yes this is possible.
you can forward the same message again & again as much time as you want in same but be careful :)

**Reply by Riders004** [reply]
How
